src/User_Command.php

```diff
@@ -498,6 +501,11 @@ public function update( $args, $assoc_args ) {
 			$user_ids[] = $user->ID;
 		}
 
+		if ( isset( $assoc_args['skip-email'] ) ) {
+			add_filter( 'send_email_change_email', '__return_false' );
+			add_filter( 'send_password_change_email', '__return_false' );
```

Comment:
Can we remove these filters after the fact, to ensure they don't bleed through to other commands?